langserver/symbol.go

```diff
@@ -59,6 +62,7 @@ func (q Query) String() string {
 	for _, token := range q.Tokens {
 		s = queryJoin(s, token)
 	}
+	s = queryJoin(s, fmt.Sprintf("CaseSensitive:%v", q.CaseSensitive))
```

Comment:
i didn't want to change the default value.
how about `CaseInsensitive` instead? defaults to false, so not included, otherwise if true, add it.